md/single-sign-on-with-saml.md

```diff
@@ -247,22 +247,33 @@ com.bonitasoft.level = ALL
 
 ## Configure logout behaviour
 
-#### Bonita Portal
-
 When your Bonita platform is configured to manage authentication over SAML, when users log out of Bonita Portal, they do not log out of the SAML Identity Provider (IdP).  
 Therefore they are not logged out of all applications that are using the IdP.  
-To avoid this, you can hide the logout option of the portal.  
-To do this, set the `logout.link.hidden=true` option in `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` 
-for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
+To avoid this, you have two options :  
+
+#### Hide the logout button of the portal
+
+This is the most commonly used solution.  
+To do this, set the `logout.link.hidden` option to `true` in `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
 
 ::: info
 **Note:** When a user logs out from the IdP, Bonita Portal's session will remain active. The user's session time to live will be reset 
-to the configured session timeout value upon each user interaction with the server.
+to the configured session timeout value upon each user interaction with the server.  
 :::
 
+#### Setup Bonita platform for SAML global logout
+
+Global logout allows to log out from the Identity Provider as well as all the registered Service Providers when logging out from Bonita platform. This is sometimes required for example if users are on public computers.  
+As Identity Providers do not necessarily support single logout and have different ways of handling it (there are several SAML Single Logout methods), Bonita only offers SAML global logout as an experimental feature. Meaning that this feature has only been tested with Keycloack server acting as Identity Provider.  
+Therefore, there is no guaranty that the global logout will work with your Identity Provider. However, if it supports the Service Provider initiated flow of the Web Browser Single Logout profile, it is likely to work.  
```

Comment:
Is there any means of explaining the condition under which it is likely to work?

Reply:
if it supports the Service Provider initiated flow of SAML's Web Browser Single Logout profile (like Keycloak does).
I will try to rephrase it.